.github/actions/setup-android/action.yml

```
@@ -0,0 +1,40 @@
+name: 'Setup Android Environment'
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)